src/scrcmd.c

```diff
@@ -2217,7 +2217,7 @@ bool8 ScrCmd_checkmonobedience(struct ScriptContext *ctx)
 
 bool8 ScrCmd_cmdCF(struct ScriptContext *ctx)
```

Comment:
Can you document this function?  It appears `gotoram` is misnamed, and should be something like `returnram`.

Reply:
Resolved in e30ea04